sites/public/src/ListingView.tsx

```diff
@@ -139,6 +139,21 @@ export const ListingView = (props: ListingProps) => {
     } else return t("listings.reservedCommunityTitleDefault")
   }
 
+  const shouldShowFeaturesDetail = () => {
```

Comment:
Does this take into account the additional fees section?

Reply:
Good catch, added that in.